tests/particles/test_particle_class.py

```diff
@@ -1695,3 +1696,12 @@ def test_isotope_ion_electron_binding_energy() -> None:
     assert C_12.electron_binding_energy == C_13.electron_binding_energy
     assert C_12.electron_binding_energy == C_14.electron_binding_energy
     assert C_13.electron_binding_energy == C_14.electron_binding_energy
+
+def test_infnite_ionization() -> None:
```

Comment:
just a little thing I noticed: infinite is spelled wrong; if the function is supposed to be called more than this time, I think it would be good to correct the spelling issue.